index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="author" content="Kaboha Jean Mark">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Next Web</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="zero-nav-line">
+                <div class="links-left">
+                    <a href="#"><span class="white-text">News</span></a>
+                    <a href="">Events</a> <a href="">Business</a> 
+                    <a href="">AMAs</a> <a href="">Spaces</a>
+                    <a href="">Terms & Condition</a></div>
+                <div class="links-right">
+                    <a href="#">About</a>
+                    <a href="">Advertise</a>
+                    <a href="">Jobs</a>
+                    <a href="">Contact</a>
+                </div>
+           </div>
+            <div class="first-nav-line">
+                <span class="logo-image"><img src="./images/logo.png" alt="logo"></span>
+                <span class="invisible-links"> <a href="#">LATEST</a> <a href="">HARD FORK</a> <a href="">PLUGGED</a> <a href="">README</a> <a href="">GROWTHQUARTERS</a></span>
+                <span><a href=""><i class="fas fa-search fa-right fa-1.5x"></i></a><a href=""><i
+                            class="fas fa-bars fa-1.5x hide-icon"></i></a></span>
+            </div>
+            <div class="second-nav-line">
+                <p>Calling young digital talent in the Netherlands. be recognzed in the T500 <i
+                        class="fas fa-arrow-right"></i></p>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="highlights">
+            <div class="highlight-1 highlight-item">
+                <div class="top-left-image">
+                    <img src="images/tnw.svg" alt="logo">
+                </div>
+                <div class="top-right-time">
+                    <p>Tuesday - January 26, 2021</p>
+                </div>
+                <div class="lower-moon">
+                    Opal cuts of your apps' internet access to help you stop doomscrolling
```

Comment:
In the Highlights (top 3 articles) section, 
- The title of each article should use a header (h1...h6) tag and has a link (a tag). Kindly make the changes. i.e.
```html
<h1>
  <a href="#">
    Opal cuts of your apps' internet access to help you stop doomscrolling
  </a>
</h1>
```